index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="style.css">
```

Comment:
I think it would be better if the head begins with the <meta> tag, it's better to put the CSS link below the <meta> tag.

Reply:
@kidde60 it would be better to explain why you require changes, maybe it's obvious for you but not for everyone